amenity-points.mss

```diff
@@ -2328,63 +2328,79 @@
     }
   }
 
-  // potentially larger offices
-  [zoom >= 17] {
-    [feature = 'office_administrative'],
-    [feature = 'office_adoption_agency'],
-    [feature = 'office_educational_institution'],
-    [feature = 'office_employment_agency'],
-    [feature = 'office_energy_supplier'],
-    [feature = 'office_financial'],
-    [feature = 'office_government'],
-    [feature = 'office_newspaper'],
-    [feature = 'office_ngo'],
-    [feature = 'office_political_party'],
-    [feature = 'office_quango'],
-    [feature = 'office_religion'],
-    [feature = 'office_research'],
-    [feature = 'office_tax'],
-    [feature = 'office_telecommunication'],
-    [feature = 'office_water_utility'],
-    {
-      text-name: "[name]";
-      text-size: @standard-font-size;
-      text-wrap-width: @standard-wrap-width;
-      text-line-spacing: @standard-line-spacing-size;
-      text-dy: 8;
-      text-fill: @office;
-      text-face-name: @standard-font;
-      text-halo-radius: @standard-halo-radius;
-      text-halo-fill: rgba(255, 255, 255, 0.6);
-      text-placement: interior;
+  [feature = 'office'] {
+    // potentially larger offices
+    [zoom >= 17] {
+      [office = 'administrative'],
```

Comment:
Wouldn't this be prone to combinatory explosion problems (see https://github.com/mapbox/carto/issues/20)? This was the reason why `feature = office_` was introduced in the first place.